physionet-django/console/templates/console/user_management.html

```diff
@@ -111,21 +111,21 @@ <h3>Profile</h3>
     </div>
   {% endfor %}
 
-  {% if groups %}
-      <div class="row mb-1">
-        <div class="col-md-3">
-          User Groups:
-        </div>
-        <div class="col-md-9">
-          {% for group in groups %}
-              <a href="{% url 'user_group' group.name %}">{{ group.name }}</a>
-          {% empty %}
-            N/A
-          {% endfor %}
-        </div>
+  <br>
```

Comment:
Doesn't really matter, but nice to use `<br />` for consistency with other breaks in the document?